source/slang/slang-ir-explicit-global-context.cpp

```diff
@@ -12,8 +12,108 @@ namespace Slang
 // thread-group, and wrap them up in an explicit "context"
 // type that gets passed between functions.
 
+enum class HoistableTypes : UInt
```

Comment:
This enum seems duplicate to the existing enum, `GlobalObjectKind`.
And the use of the word, "type", here is confusing because it is not type like `int`, `float` and so on.

Reply:
I agree, I should just be using `GlobalObjectKind`.